rest/system-pause-post.rst

```diff
@@ -1,6 +1,8 @@
 POST /rest/system/pause
 =======================
 
-Pause the given device.
+Pause the given device or all devices.
 
-Takes the mandatory ``device`` parameter and returns status 200 and no content upon success, or status 500 and a plain text error on failure.
+Takes the optional parameter ``device`` (device name). When ommitted,
```

Comment:
It is actually the device name (as in UI, definitely better than device string), not the device ID. See:
https://github.com/syncthing/syncthing/blob/master/cmd/syncthing/gui.go#L1130